cosqa_data_code.py

```diff
@@ -0,0 +1,102 @@
+""" This module houses code to generate relevances for the cosQA data.
+    This data serves as a comparison and benchmark for our works.
+    This module is a 'data shim', it sits between the format of cosQA that is publicly available
+    and the format we use in this project for annotations of code search.
+    Once this module is executed, the expected output is a csv file whose
+    subsequent use should work 'as-is' as an input for the `gen-llm-rels` click command.
+    Then once those relevances are determined we can use them for metrics calculation in the
+    `gen-ir-metrics` click command. Therefore, we use this module only once in the workflow for
+    comparing the results.
```

Comment:
```suggestion
    comparing the results. The paper that describes the cosQA data is here: https://arxiv.org/abs/2105.13239
```

@D-Roberts does this work? 

This is the only code-search benchmark dataset I've been able to find that has actual query relevances in the data.
So I didn't really feel like a list would make sense. I plan to add some writeup in the paper on this data though.